Library/Koala/Koala.swift

```diff
@@ -1657,6 +1657,24 @@ public final class Koala {
     self.track(event: "Closed Live Stream", properties: props)
   }
 
+  public func trackLiveStreamApiErrorOccurred(project: Project,
+                                              liveStreamEvent: LiveStreamEvent,
+                                              error: LiveApiError) {
+    let props = properties(project: project, loggedInUser: self.loggedInUser)
+      .withAllValuesFrom(properties(liveStreamEvent: liveStreamEvent))
+      .withAllValuesFrom(
+        [
+          "context": stateContext(forLiveStreamEvent: liveStreamEvent).trackingString,
+          "error": error
+        ]
+    )
+
+    self.track(
+      event: "Live API Error",
```

Comment:
We can remove that but there isn't anything else to do with these errors, nothing user-actionable so I didn't want to show an alert but I figured we may want to know if a lot of these are happening over time. Not sure if it is trackable from api side.

PR card added for other events.